non_www/tabs/school/download_national.html

```diff
@@ -9,33 +9,34 @@
+        <p>Please be aware that equivalent files for different purposes and geographies have <b>the same file name</b> when downloaded (e.g. 'z_all.geojson' for national zones files for both commuting and school travel).  To avoid confusion you may want to rename those files or save them in labelled folders. Note also that when <b>downloading 'csv' files using a Firefox browser</b>, the file will open onscreen in a new webpage.  If you then type "Ctrl + S" the onscreen data will download as a csv file.</p>
       </div>
 
       <div class="col-md-12 text-left" style="padding-top:10px;">
         <h4><a name="natschoolzones"></a>School travel data for zones</h4>
-        <p>Download anonymised National School Census 2011 baseline data and scenario attribute data for all zones in England. These files present aggregate data for all children <b>living in a given zone</b>. </p>
+        <p>Download anonymised National School Census 2011 baseline data and scenario attribute data for all zones in England. These files present aggregate data for all children <b>living in a given zone</b></p>
+        <p>In the LSOA downloads, baseline and scenario data are not presented for cells containing 1 or 2 children.  Instead these data are set as missing ('NA').  These data are, however, included in the local authority downloads.</p>
         <ul>
           <li>Zones (local authorities)*:
-            <a href="https://github.com/npct/pct-outputs-national/raw/master/school//lad_attributes.csv" target="_blank" download="lad_attributes.csv" onclick="ga('send', 'event', 'download', 'lad_attributes.csv');">
+            <a href="https://github.com/npct/pct-outputs-national/raw/master/school_public//lad_attributes.csv" target="_blank" download="lad_attributes.csv" onclick="ga('send', 'event', 'download', 'lad_attributes.csv');">
               csv
             </a>
-            <a href="https://github.com/npct/pct-outputs-national/raw/master/school//lad.Rds" target="_blank" download="lad.Rds" onclick="ga('send', 'event', 'download', 'lad.Rds');">
+            <a href="https://github.com/npct/pct-outputs-national/raw/master/school_public//lad.Rds" target="_blank" download="lad.Rds" onclick="ga('send', 'event', 'download', 'lad.Rds');">
               Rds
             </a>
-            <a href="https://github.com/npct/pct-outputs-national/raw/master/school//lad.geojson" target="_blank" download="lad.geojson" onclick="ga('send', 'event', 'download', 'lad.geojson');">
+            <a href="https://github.com/npct/pct-outputs-national/raw/master/school_public//lad.geojson" target="_blank" download="lad.geojson" onclick="ga('send', 'event', 'download', 'lad.geojson');">
```

Comment:
Very minor, I missed it last time but the URLs above have two `//`, i.e.
```html
school_public//lad.geojson
```
it shouldn't effect anything but might on some strange browser.

also, I'm assuming that we are going to put the schools data in `school_public` in the repos?

Reply:
The same `//` exists in https://github.com/npct/pct-shiny/blob/master/non_www/tabs/commute/download_national.html which is probably my fault. Sorry!

Reply:
Might also be nice to separate these Google Analytics events out.  We could use:
```
onclick="ga('send', 'event', 'download', 'school_public/lad.geojson')
```

and same with all the other events.  I'm happy to make that change if you like.

Reply:
I think the // is a legacy from when we used a function to generate those URLs, and the lad. files are one level up from the other ones. I have now replaced this.

good idea in terms of Google analytics – I have made the change I think you mean